### DISCUSSION TITLE
Support for other external shares

### DISCUSSION THREAD

**Discussion by RailsViceRoads** [JonathanTreffler/sharerenamer#80] [Ideas]
Thank you for reactivating  this app.

Is it possible to extend the app to modify other external shares? In my case it will be mainly the Polls application and some times the Forms app.

**Comment by welaq** [comment]
In my case I am wondering whether it'd be possible to make it work with Calendar app.

**Comment by y8s** [comment]
And [Memories](https://apps.nextcloud.com/apps/memories). Would love pretty URLs for shared albums.
